inst/tests/tests.Rraw

```diff
@@ -18447,25 +18451,36 @@ test(2253.05, options=list(datatable.prettyprint.char = 3L), DT, output=paste0(s
 test(2253.06, options=list(datatable.prettyprint.char = 1L), DT, output=paste0(strrep(ja_ichi, 1L), dots))
 # Tests for multiple, different length combining character rows
 DT = data.table(strrep(accented_a, 1L:4L))
-test(2253.07, options=list(datatable.prettyprint.char = 4L), gsub(clean_regex, "", capture.output(print(DT))[-1L]), c("á", "áá", "ááá", "áááá"))
-test(2253.08, options=list(datatable.prettyprint.char = 3L), gsub(clean_regex, "", capture.output(print(DT))[-1L]), c("á", "áá", "ááá", "ááá..."))
-test(2253.09, options=list(datatable.prettyprint.char = 1L), gsub(clean_regex, "", capture.output(print(DT))[-1L]), c("á", "á...", "á...", "á..."))
+test(2253.07, options=list(datatable.prettyprint.char = 4L), gsub(clean_regex, "", capture.output(print(DT))[-1L]), strrep(accented_a, 1:4L))
```

Comment:
`strrep()` is from R 3.3.0, so we can't use it so easily yet.

Let's optionally define an equivalent here instead:

```r
if (!exists("strrep", "package:base")) {
  strrep = function(x, times) mapply(function(x, times) paste(rep(x, times), collapse=""), rep_len(x, length(times)), times, USE.NAMES=FALSE)
}
```

Reply:
Should I put this at the beginning of the tests or near the beginning of the entire tests.Rraw file? AFAICT my tests aren't the only ones calling `strrep()`

Reply:
Good point, yes put it at the top.